presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/AgentStatusUtils.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.teradata.prestomanager.agent;
+
+import javax.ws.rs.core.Response;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.TimeUnit;
+
+import static com.google.common.util.concurrent.Uninterruptibles.sleepUninterruptibly;
+import static com.teradata.prestomanager.agent.AgentFileUtils.getFileProperty;
+import static com.teradata.prestomanager.agent.CommandExecutor.executeCommand;
+import static com.teradata.prestomanager.common.RequestBuilder.newRequestBuilder;
+import static java.lang.String.format;
+import static javax.ws.rs.client.Entity.json;
+import static javax.ws.rs.core.Response.Status.OK;
+
+// TODO: stop presto before upgrade and uninstall
+public final class AgentStatusUtils
+{
+    private static final long WAIT_TIME_SECONDS = 60;
+    private static final long STEP_MILLISECONDS = 100;
+    private static final String LAUNCHER_SCRIPT = "/usr/lib/presto/bin/launcher";
+    private static final Path PATH_TO_CONFIG_FILE = Paths.get("/etc/presto/config.properties");
+
+    private AgentStatusUtils() {}
+
+    public static void startPresto()
+            throws PrestoManagerException
+    {
+        if (executeCommand("rpm -q presto-server-rpm") != 0) {
+            throw new PrestoManagerException(format("Presto not installed"));
+        }
+        if (executeCommand("service presto status") == 0) {
+            throw new PrestoManagerException("Presto is already running");
+        }
+        if (executeCommand("service presto start") != 0) {
+            throw new PrestoManagerException("Failed to start presto");
+        }
+        long endTime = System.currentTimeMillis() + TimeUnit.SECONDS.toMillis(WAIT_TIME_SECONDS);
```

Comment:
This part of the code is to periodically check whether presto is installed or not.
There's no standard lib to do this stuff.